pabi_hr_expense_report/pabi_dunning_letter/wizard/pabi_dunning_letter_wizard.xml

```diff
@@ -20,18 +20,18 @@
                         <group>
                             <field name="print_pdf"/>
                             <field name="send_email"/>
-                            <field name="to_mail_group_ids" widget="many2many_tags"/>
```

Comment:
I think we should still have TO and CC email as many2many to show user who will be sent to.
(we create a database view model for the email id, based on hr_employee?)
For the group email, you can have separated field, I have no issue since I still not sure also when should we keep that mail.